custom.css

```diff
@@ -6,7 +6,7 @@
   margin: 0;
   box-sizing: border-box;
   font-family: 'Poppins', sans-serif;
-  overflow-y: hidden;
+  overflow-x: hidden;
```

Comment:
- [ ] As requested by previous reviewers, this overflow did not work as expected, it prevent your app from scrolling instead, please see what your modifications brought.

![6](https://user-images.githubusercontent.com/77843050/231470128-c6b1d88d-fb4b-4e1a-a376-daa50a9999b4.gif)